docs/sources/tempo/metrics-from-traces/metrics-queries/sampling-guide.md

```diff
@@ -0,0 +1,950 @@
+- Uses probabilistic sampling with reservoir sampling for quantile calculations
+- Applies back pressure when downstream processing becomes a bottleneck
+- Scales sampling factors based on actual data encountered, not estimated volumes
+
+#### Limitations
+
+- May over-sample rare events, reducing performance benefits for needle-in-haystack queries
+- Introduces slight non-determinism in results across repeated executions
+- Less suitable for precise alerting scenarios requiring consistent thresholds
+
+### Fixed span sampling: `with(span_sample=0.xx)`
+
+Fixed span sampling provides deterministic control over sampling rates by selecting a specified percentage of spans regardless of query characteristics. This approach offers predictable performance improvements and consistent approximation levels across different queries.
+
+The span sampler applies probabilistic sampling at the span level using consistent hashing of span IDs. This ensures reproducible sampling decisions—the same span is consistently included or excluded across different query executions. The sampling occurs early in the query pipeline, reducing I/O and processing overhead.
+
+#### Syntax and examples
+
+```traceql
+// Sample 10% of spans for error rate analysis
+{ status=error } | rate() by (resource.service.name) with(span_sample=0.1)
+
+// Sample 5% of spans for latency percentiles
+{ span.http.method="GET" } | quantile_over_time(duration, .95) with(span_sample=0.05)
+
+// Sample 1% of spans for high-level service overview
+{ } | count_over_time() by (resource.service.name) with(span_sample=0.01)
+```
+
+#### Best use cases
+
+- **Consistent approximation** when you need predictable accuracy levels
+- **Large-scale monitoring** where 1-5% sampling provides sufficient signal
+- **Baseline metrics** for capacity planning and trend analysis
+- **Cost optimization** in cloud environments where query costs scale with data processed
+
+#### Technical characteristics
+
+- Deterministic sampling based on span ID hashing
+- Linear performance scaling with sampling percentage
+- Preserves temporal distribution of spans
+- Compatible with all TraceQL metrics functions
+
+#### Limitations
+
+- Fixed sampling may miss important events during low-volume periods
+- Not optimal for queries with naturally low selectivity
+- May under-represent short-lived or infrequent services
+
+### Fixed trace sampling: `with(trace_sample=0.xx)`
+
+Fixed trace sampling operates at the trace level, selecting complete traces for analysis rather than individual spans. This approach is particularly effective for queries that require full trace context or perform trace-wide aggregations.
+
+Trace sampling uses consistent hashing of trace IDs to make sampling decisions, ensuring that once a trace is selected, all spans within that trace are included in the analysis. This preserves trace integrity and maintains accurate relationships between spans within the same request flow.
```

Comment:
This part is also incorrect. There is no relation to consistent hashing for sampling.  It sounds like it's describing the general concept of tail sampling.

Reply:
Removed in updated version.